prow/spyglass/lenses/links/links_test.go

```diff
@@ -55,3 +59,40 @@ func TestHumanReadableName(t *testing.T) {
 		})
 	}
 }
+
+func TestToLink(t *testing.T) {
+	spyglassConfig := config.Spyglass{
+		GCSBrowserPrefix: "http://gcsbrowser/",
+	}
+	tests := []struct {
+		jobPath string
+		content []byte
+		want    link
+	}{
+		{
+			jobPath: "artifacts/dashboard.txt",
+			content: []byte("   http://website.com/asdasds?adssasds   "),
+			want: link{
+				Name: "Dashboard",
+				URL:  "http://website.com/asdasds?adssasds",
+				Link: "https://google.com/url?q=http%3A%2F%2Fwebsite.com%2Fasdasds%3Fadssasds",
+			},
+		},
+		{
+			jobPath: "artifacts/master-and-node-logs.txt",
+			content: []byte("gs://bucket/asdasd/asdasd\n"),
```

Comment:
Wouldn't more likely content be something like:

```
foo.go:38 did something
foo.go:100 did something else see http://www.google.com for more info
foo.go:38 did something
```

Reply:
The expected content is either gs:// path or link. Example here: https://prow.k8s.io/view/gs/kubernetes-jenkins/logs/ci-kubernetes-e2e-gce-scale-performance/1401947321612111872